CHANGELOG.md

```diff
@@ -17,12 +17,15 @@ All notable changes to this project will be documented in this file.
 - [#1112](https://github.com/pints-team/pints/pull/1112) A new `NoUTurnMCMC` sampler (NUTS) was added, along with a `DualAveragingAdaption` class to adaptively tune related Hamiltonian Monte Carlo methods.
 - [#1025](https://github.com/pints-team/pints/pull/1025) Added a stochastic logistic growth problem for use with ABC.
 ### Changed
+- [#1250](https://github.com/pints-team/pints/pull/1250) The returned values from `SingleChainMCMC.tell()` and `MultiChainMCMC.tell()` have been extended from current position `x` to `x, fx, accepted`, where `fx` is the current log likelihood and `accepted` is a bool indicating whether tell performed an acceptance step in this call.
```

Comment:
Do we want to say we've changed the version to 0.4.0 here as well...?

Reply:
Maybe!
But I think we release version 0.4.0, and then people go look to see what's new, so then they don't need to see "it's 0.4.0 now" again :D